lib/auth/webauthnwin/api.go

```diff
@@ -188,7 +190,12 @@ type CheckSupportResult struct {
 // false positives.
 // See CheckSupport.
 func IsAvailable() bool {
-	return CheckSupport().IsAvailable
+	supports := CheckSupport()
+	if supports.HasCompileSupport && !supports.IsAvailable {
+		log.Warn("WebAuthnWin: not supported on your windows version, supported from version 1903")
```

Comment:
Should we wrap this in a sync.Once to avoid spamming if the flow changes?

Reply:
i was thinking about that, but execution via tsh will mostly call it just once, because we execute single command and exit, right?

Reply:
I think so too. Your call.